### DISCUSSION TITLE
Regarding Analytics API

### DISCUSSION THREAD

**Discussion by iamhrugved** [vercel/community#2746] [Help]
### Summary

I have built a portfolio website and deployed it to Vercel. I want to get the Analytics data, like visits and clicks on my website through API, and display it on my webpage, but I couldn't find the exact API endpoint to get this data in the docs. Is it possible to do so? If yes, can anyone please help me with this?

### Steps to Reproduce

const res = await fetch(
      `https://api.vercel.com/v9/projects/${process.env.NEXT_PUBLIC_VERCEL_PROJECT_ID}/analytics/summary`,
      {
        headers: {
          Authorization: `Bearer ${process.env.NEXT_PUBLIC_VERCEL_TOKEN}`,
        },
      }
    );
    
    I tried this API, but it's giving 404

**Comment by swarnava** [comment]
Hi,

You should use Speed Insights API https://vercel.com/docs/concepts/speed-insights/api

**Reply by iamhrugved** [reply]
The Speed Insights API doesn't allow me to get the data for Analytics, like Visitors and Page Views. How can I access that?
